pwndbg/typeinfo.py

```diff
@@ -36,22 +36,31 @@ def is_pointer(value):
 @pwndbg.events.start
 @pwndbg.events.stop
 def update():
+
+    def lookup_first_type(types):
+        for type_str in types:
+            try:
+                return gdb.lookup_type(type_str)
+            except:
+                pass
+        raise Exception('Failed to lookup types: {}'.format(types))
+
     module.char   = gdb.lookup_type('char')
-    module.ulong  = gdb.lookup_type('unsigned long')
-    module.long   = gdb.lookup_type('long')
-    module.uchar  = gdb.lookup_type('unsigned char')
-    module.ushort = gdb.lookup_type('unsigned short')
-    module.uint   = gdb.lookup_type('unsigned int')
+    module.ulong  = lookup_first_type(('unsigned long', 'uint'))
+    module.long   = lookup_first_type(('long', 'int'))
+    module.uchar  = lookup_first_type(('unsigned char', 'ubyte'))
+    module.ushort = lookup_first_type(('unsigned short', 'ushort'))
+    module.uint   = lookup_first_type(('unsigned int', 'uint'))
     module.void   = gdb.lookup_type('void')
-    module.uint8  = gdb.lookup_type('unsigned char')
-    module.uint16 = gdb.lookup_type('unsigned short')
-    module.uint32 = gdb.lookup_type('unsigned int')
-    module.uint64 = gdb.lookup_type('unsigned long long')
+    module.uint8  = module.uchar
+    module.uint16 = module.ushort
+    module.uint32 = module.uint
+    module.uint64 = lookup_first_type(('unsigned long long', 'ulong'))
 
     module.int8   = gdb.lookup_type('char')
     module.int16  = gdb.lookup_type('short')
     module.int32  = gdb.lookup_type('int')
-    module.int64  = gdb.lookup_type('long long')
+    module.int64  = lookup_first_type(('long long', 'long'))
```

Comment:
Nevermind, after looking at the `ctod` docs, I see that D defines these differently.